doc/js_tutorials/js_core/js_basic_ops/js_basic_ops.markdown

```diff
@@ -0,0 +1,293 @@
+Basic Operations on Images {#tutorial_js_basic_ops}
+==========================
+
+Goal
+----
+
+Learn to:
+
+-   Access image properties
+-   How to construct Mat
+-   How to copy Mat
+-   How to convert the type of Mat
+-   Access pixel values and modify them
+-   Setting Region of Interest (ROI)
+-   Splitting and Merging images
+
+Accessing Image Properties
+--------------------------
+
+Image properties include number of rows, columns and size, depth, channels, type of image data.
+
+@code{.js}
+let src = cv.imread("canvasInput");
+let rows =  src.cols;
+let columns = src.rows;
+let size = src.size;
+let depth = src.depth();
+let channels = src.channels();
+let type = src.type();
+@endcode
+
+@note src.type() is very important while debugging because a large number of errors in OpenCV-Python
+code is caused by invalid data type.
+
+How to construct Mat
+------------------------------------
+
+There are 4 constructors:
+
+@code{.js}
+let mat = new cv.Mat(size, type);                        // 2 parameters
+let mat = new cv.Mat(rows, cols, type);                  // 3 parameters
+let mat = new cv.Mat(rows, cols, type, new cv.Scalar()); // 4 parameters
+let mat = matFromArray(rows, cols, type, array);
+@endcode
+
+There are 3 static functions:
+
+@code{.js}
+let mat = cv.Mat.zeros(rows, cols, type);           
+let mat = cv.Mat.ones(rows, cols, type); 
+let mat = cv.Mat.eye(rows, cols, type);
+@endcode
+
+**warning**
+
+Don't forget to delete cv.Mat(cv.MatVector) when you don't want to use it any more.
+
+How to copy Mat
+------------------------------------
+
+@code{.js}
+dst = src.clone();
+src.copyTo(dst, mask); // only entries indicated in the arry mask are copied    
```

Comment:
You'd better to split the two copy methods into two examples, otherwise people will think they are two steps to copy a mat.